features/step_definitions/test_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^I visit the menu page$/) do
+  visit '/menu'
+end
+
+Then(/^I should see "([^"]*)"$/) do |content|
+  expect(page).to have_content content
+end
+
+Then(/^I should see "([^"]*)" with "([^"]*)"$/) do |name, price|
+  expect(page).to have_content name
+  expect(page).to have_content price
+end
+
+Given(/^the following dishes exist$/) do |table|
+  table.hashes.each do | dish |
+  Dish.create(dish)
+end
+end
```

Comment:
Formatting here is a problem - if you ever have two `end` statements stacked on each other like this, it means there is an indentation problem. This block should look like:
```
Given(/^the following dishes exist$/) do |table|
  table.hashes.each do | dish |
    Dish.create(dish)
  end
end
```
See how that demonstrates that the `Dish.create(dish)` is *inside* the `table.hashes` block? That's what we mean about "readability"